bouncer/scripts/test/redirect-test.js

```diff
@@ -110,6 +111,24 @@ describe('redirect()', function () {
     assert.equal(console.error.called, true);
   });
 
+  it('calls chrome.runtime.sendMessage correctly', function () {
+    window.chrome = {
+      runtime: {
+        sendMessage: sinon.stub()
+      }
+    };
+
+    redirect(function () {});
+
+    assert.equal(window.chrome.runtime.sendMessage.calledOnce, true);
```

Comment:
The problem is the third argument is an inline function from `redirect.js` though